modular_skyrat/modules/modular_ert/code/trauma_team/truama_team_outfit.dm

```diff
@@ -26,16 +60,47 @@
 	new /obj/item/blood_filter(src)
 	new /obj/item/healthanalyzer/advanced(src)
 
+/obj/item/storage/firstaid/tactical/ntrauma
+	name = "trauma medical kit"
+	desc = "I hope you've got insurance, because the Trauma Team's premiums are HIGH."
 
 /obj/item/storage/firstaid/tactical/ntrauma/PopulateContents()
 	new /obj/item/stack/medical/gauze(src)
+	new /obj/item/stack/medical/splint(src)
 	new /obj/item/defibrillator/compact/combat/loaded/nanotrasen(src)
 	new /obj/item/reagent_containers/pill/patch/libital(src)
 	new /obj/item/reagent_containers/pill/patch/libital(src)
 	new /obj/item/reagent_containers/pill/patch/aiuri(src)
 	new /obj/item/reagent_containers/pill/patch/aiuri(src)
-	new /obj/item/clothing/glasses/hud/health/night(src)
+	new	/obj/item/holosign_creator/medical (src)
+	new /obj/item/pinpointer/crew/prox (src)
+
+/obj/item/clothing/gloves/color/latex/nitrile/ntrauma
+	name = "trauma specialist gloves"
+	desc = "A pair of nitrile-alternative gloves used by Trauma Team specialists, with a unique (and expensive) acid-repellent coating to prevent damage handling chemical hazards. Wont protect the rest of your body, though."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/gloves.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/hands.dmi'
+	icon_state = "ert_ntrauma"
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+
+/obj/item/clothing/suit/space/ntrauma
+	name = "trauma team softsuit"
+	desc = "A lightweight, minimally armored, entirely sterile softsuit, used by Trauma Teams to operate in potentially hazardous environments of all sorts. It's coated in acid-repellent chemicals."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	icon_state = "ert_ntrauma"
+	permeability_coefficient = 0.01
+	slowdown = 0.3
+	armor = list(MELEE = 10, BULLET = 10, LASER = 10,ENERGY = 10, BOMB = 10, BIO = 100, FIRE = 80, ACID = 80)
+	resistance_flags = ACID_PROOF
+	cell = /obj/item/stock_parts/cell/super
+	mutant_variants = NONE	//Traumateam NEEDS to look nondescript. Its the whole gimmick, tactical healing.
 
-/obj/item/clothing/gloves/color/latex/nitrile/infiltrator/ntrauma
```

Comment:
Infiltrator Gloves increase grabbing speed, vital for fireman carrying the VIP out

Reply:
These werent infiltrator gloves, they were a whole new path that doesnt exist:

`/obj/item/clothing/gloves/color/infiltrator` are the normal infiltrator gloves, which have none of the latex bio protection.
Furthermore, Latex applies the same buff, TRAIT_QUICKER_CARRY